File: 8KYU/Compare with margin/solution.js
function closeCompare(a, b, margin=0){
    console.log(`${a}, ${b}, ${margin}`);
    if(Math.abs(a-b) <= margin)
      return 0;
    else if(a > b)
      return 1;
    else
      return -1;
  }
  

File: 8KYU/Compare with margin/solution.go
package kata

import (
	"math"
)

func CloseCompare(a, b, margin float64) int {

	if math.Abs(a-b) <= margin {
		return 0
	} else if a > b {
		return 1
	} else {
		return -1
	}
}
